README.md

```diff
@@ -23,7 +23,7 @@ These instructions assume that you have set up a server (either Mac or Ubuntu) w
 ### Installation
 
     npm install -g webdriverio-server
-    webdriverio-server-init
+    ./node_modules/.bin/webdriverio-server-init
```

Comment:
With the `npm install -g` (as shown in like 25), this won't work, unless you happen to be inside the root `node_modules` directory.